ansible/roles/runner/tasks/main.yml

```diff
@@ -1,10 +1,10 @@
 ---
-- include: manage_ssh_keys.yml
+- import_tasks: manage_ssh_keys.yml
```

Comment:
Trying to test it I got this error:
```
[fbarreto@~/projects/freeipa-pr-ci] (pr128) ansible-playbook -i ansible/hosts/runners ansible/prepare_devel_test_runners.yml

ERROR! no action detected in task. This often indicates a misspelled module name, or incorrect module path.

The error appears to have been in '/home/fbarreto/projects/freeipa-pr-ci/ansible/roles/runner/tasks/main.yml': line 2, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

---
- import_tasks: manage_ssh_keys.yml
  ^ here


The error appears to have been in '/home/fbarreto/projects/freeipa-pr-ci/ansible/roles/runner/tasks/main.yml': line 2, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

---
- import_tasks: manage_ssh_keys.yml
  ^ here
```

```
$ ansible --version
.ansible 2.3.1.0
```

Am I missing something here?

Reply:
Ugh, you're not. This syntax is only supported in 2.4+, yet it already complains it's deprecated...